core/templates/dev/head/pages/exploration_player/supplemental_card_directive.html

```diff
@@ -21,6 +21,15 @@
   <div class="protractor-test-conversation-input"
        angular-html-bind="activeCard.interactionHtml">
   </div>
+    <div ng-if="currentInteractionHints.length" style="text-align: right; padding: 8px">
+      <span tooltip="<[!isHintAvailable() ? 'Please try answering the question first.' : 'Click here for a hint!']>">
+        <md-button class="hint-button"
+                   ng-click="showHint()"
+                   ng-disabled="!isHintAvailable()">
```

Comment:
I noticed that at one point I, being clumsy, accidentally tried submitting an answer by clicking the hint button instead of the submit button--it's a very tempting button to press as it's very dark/bold compared to the white submit button, and it's in a pretty convenient position, and it's in all CAPS which makes it stand out more. Most people probably aren't as clumsy as I am, but I feel there will be instances where people make the same mistake. Maybe if the user has text in the response field, disable the hint button and have the tooltip be something like "Try submitting what you've typed first"